V1/DTOs/Wallet/WalletDetailRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CoviIDApiCore.V1.DTOs.Credentials;
+using System.ComponentModel.DataAnnotations;
+
+namespace CoviIDApiCore.V1.DTOs.Wallet
+{
+    public class WalletDetailRequest
+    {
+        [StringLength(50)]
+        public string FirstName { get; set; }
+        [StringLength(50)]
+        public string LastName { get; set; }
+        public string Photo { get; set; }
+        public string MobileNumber { get; set; }
```

Comment:
Maybe some regex validation on this. Or at minimum length validation (no shorter than 10 and not longer than 14)

Reply:
(I mentioned this above)

I think MobileNumber shouldn't be in WalletDetail at all - only on Wallet.

Reply:
Yes, I believe these changes were made to the database design after she implemented in code, so just need to be updated.